packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx

```diff
@@ -371,8 +638,55 @@ const GivingSeasonHeader = ({
                   )}
                 </div>
               )}
-              {rightHeaderItems}
+              <div className={classes.gsRightHeaderItems}>
+                {rightHeaderItems}
+              </div>
             </Toolbar>
+            <div className={classes.gsContent}>
+              <div>
+                <span>Get your 2023 donations in</span>
+              </div>
+              <div className={classes.gsContentSubtitle}>
+                {currentUser
+                  ? (
+                    <span>
+                      Forum users who already donated: consider adding a heart
+                      by clicking on this banner.
+                    </span>
+                  )
+                  : (
+                    <span>
+                      Explore the funds below — or log in to add a heart if
+                      you’ve already donated.
+                    </span>
+                  )
+                }
+              </div>
+              <div className={classes.gsButtons}>
+                <EAButton href="https://funds.effectivealtruism.org/funds/animal-welfare">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Animal Welfare Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/far-future">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Long-term Future Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/global-development">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Global Development Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/ea-community">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  EA Infrastructure Fund
+                </EAButton>
```

Comment:
I'd also suggest making these icons the same color as the text (dark red) and maybe a bit smaller, and also adding a bit more space above this row of buttons